test/core/messages.integration.test.ts

```diff
@@ -141,6 +141,72 @@ describe('messages integration', () => {
     ]);
   });
 
+  it<TestContext>('should be able to update and receive update messages', async (context) => {
+    const { chat } = context;
+
+    const room = getRandomRoom(chat);
+
+    // Attach the room
+    await room.attach();
+
+    // Subscribe to messages and filter them when they arrive
+    const messages: Message[] = [];
+    const updates: Message[] = [];
+    room.messages.subscribe((messageEvent) => {
+      switch (messageEvent.type) {
+        case MessageEvents.Created: {
+          messages.push(messageEvent.message);
+          break;
+        }
+        case MessageEvents.Updated: {
+          updates.push(messageEvent.message);
+          break;
+        }
+        default: {
+          throw new Error('Unexpected message event type');
+        }
+      }
+    });
+
+    // send a message, and then delete it
+    const message1 = await room.messages.send({ text: 'Hello there!' });
+    await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
What is the need for the timeout here?

Reply:
I thought to allow some time for the `createdAt` and `updatedAt` to surely be different, but they will be anyway. I'll remove this line